d2compiler/compile.go

```diff
@@ -746,6 +746,14 @@ func flattenContainer(g *d2graph.Graph, obj *d2graph.Object) {
 			newEdge, _ = g.Root.Connect(e.Src.AbsIDArray(), obj.AbsIDArray(), e.SrcArrow, e.DstArrow, e.Attributes.Label.Value)
 		}
 		// TODO more attributes
+		if e.SrcTableColumnIndex != nil {
+			newEdge.SrcTableColumnIndex = new(int)
+			*newEdge.SrcTableColumnIndex = *e.SrcTableColumnIndex
```

Comment:
one line `newEdge.SrcIndex = e.SrcIndex`?

Reply:
Not sure about copying the pointer around, so I decided to go with a new one as this is a new edge